apps/x509.c

```diff
@@ -584,6 +586,7 @@ int x509_main(int argc, char **argv)
             break;
         case OPT_CHECKEND:
             checkend = 1;
+            noout = 1;
```

Comment:
Why this change?

Reply:
I removed the `goto end;` from the end of the `if (checkend)` section to be able to keep it in the certificate "loop".  Adding `noout` keeps it functionally equivalent (ie. it was going to skip the output section anyway).

We CAN remove it, but we'll also have to add a `goto end` case in the loop check.